Buform.Example.Core/ComponentsViewModel.cs

```diff
@@ -298,6 +299,13 @@ public ComponentsViewModel(ILoggerFactory logFactory, IMvxNavigationService navi
                         Label = "Time",
                         InputType = DateTimeInputType.Time
                     }
+                },
+                new TextFormGroup("Digit generation")
+                {
+                    new DigitGenerationItem(() => GeneratedDigit)
```

Comment:
I would call this item RandomNumberGenerator or something like that, as Generation not so much info about its functionality.